homeassistant/util/unit_system.py

```diff
@@ -93,6 +96,7 @@ def __init__(
         mass: str,
         pressure: str,
         accumulated_precipitation: str,
+        length_conversions: dict[str | None, str],
```

Comment:
I think this should not be compulsory.
```suggestion
        *,
        length_conversions: dict[str | None, str] | None = None,
```

Reply:
There's no one instantiating `UnitSystem` outside util/unit_system.py, so I don't think a default value is helpful.

We should perhaps enforce kwargs for all `__init__` arguments though, it makes the instantiations much more readable.